apps/prairielearn/src/pages/instructorInstanceAdminPublishing/components/LegacyAccessRuleCard.tsx

```diff
@@ -17,7 +17,12 @@ export function LegacyAccessRuleCard({
   return (
     <div class="card mb-4">
       <div class="card-header bg-primary text-white d-flex align-items-center justify-content-between">
-        <h1>{courseInstance.long_name} course instance access rules</h1>
+        <h1>Access rules</h1>
+      </div>
+      <div class="alert alert-warning" role="alert">
```

Comment:
Minor preference to move this outside the card.

Currently:

<img width="633" height="209" alt="Image" src="https://github.com/user-attachments/assets/61642531-8639-4ea3-b96f-9fbc89e389ff" />

Proposed change:

<img width="549" height="211" alt="Image" src="https://github.com/user-attachments/assets/c123bf89-7b12-499b-a772-b412347c776e" />

Reply:
Fixed in 55c48160a88c191bdb71a899849ab38849c3267c.